### ISSUE TITLE
Question: What's the recommended approach for implementing platform-specific views (handlers)?

### ISSUE THREAD

**Issue by dszymon** [adospace/reactorui-maui#268]
I want to create a component that renders a platform-specific view. MAUI's [documentation ](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/handlers/create?view=net-maui-9.0)about handlers is helpful, but I'm unsure how to map these to Reactor's compatible format. Is `ScaffoldAttribute` recommended in this case?

**Comment by adospace** [comment]
Yes, you have first to create the custom control (test it in a normal MAUI application) then make a wrapper in MauiReactor using the Scaffold attribute (https://adospace.gitbook.io/mauireactor/components/wrap-3rd-party-controls)

**Comment by dszymon** [comment]
Thanks 👍
